src/kirby/components/modal/services/modal.controller.ts

```diff
@@ -7,11 +7,18 @@ import { ActionSheetHelper } from './action-sheet.helper';
 import { ModalConfig } from '../modal-wrapper/config/modal-config';
 import { ActionSheetConfig } from '../action-sheet/config/action-sheet-config';
 import { AlertConfig } from '../alert/config/alert-config';
+import { KirbyAnimation } from '@kirbydesign/designsystem/animation/kirby-animation';
+
+export const modalScrollPlaceholderFunction = () => {
```

Comment:
Consider naming, e.g.
```suggestion
export const modalScrollNoop = () => {
```
or
```suggestion
export const noWindowsToScroll = () => {
```